### PR TITLE
adding link & clarify text

### PR DESCRIPTION
**Pull request by jetzlstorfer**
Signed-off-by: jetzlstorfer <juergen.etzlstorfer@dynatrace.com>

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -51,7 +51,7 @@
     </div>
     <div class="row pt-3 pt-md-0">
       <div class="col-12 pt-4">
-        <p class="footer-text">Initially created by Dynatrace</p>
+        <p class="footer-text">Initially created by <a href="https://www.dyntrace.com"  class="footer-text">Dynatrace</a></p>
```
Comment (christian-kreuzberger-dtx) [inline layouts/partials/footer.html:54]: I think we should add ` target="_blank" rel="noopener"` to the link

Comment (jetzlstorfer) [reply layouts/partials/footer.html]: added it

**Comment by jetzlstorfer** [comment]
Preview https://deploy-preview-728--keptn.netlify.app/

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**